app/views/orgs/_departments.html.erb

```diff
@@ -0,0 +1,48 @@
+<%# locals: { context, title, tooltip } %>
+<% department =  department.nil? ? Department.new :  department %>
+<%= form_for(department, url: "/departments/create_or_update", html: { multipart: true, method: method, id: "edit_org_departments_form" } ) do |f| %>
+<%= f.hidden_field(:org_id, value: org.id) %>
+<%= f.hidden_field(:id, value: department.id) %>
+
+  <div class="department">
+    <div class="row">
+      <div class="col-xs-12">
+        <h3><%=_('Schools or Departments') %></h3>
+        <div class="row">
+              <div class="form-group col-xs-10">
+                <%= f.label :name, _('Name'), class: "control-label", "aria-required": true %>
+                <%= f.text_field :name, id: "org_department_name", class: "form-control" %>
```

Comment:
you shouldn't need to give it an id here. it should generate `department_name`. If all of the fields need to have the `org_` prefix on it, consider giving the form definition above a namespace instead: `<%= form_for(department, :org, url: department_path, html: { multipart: true, method: method, id: "edit_org_departments_form" } ) do |f| %>